python/requirements.txt

```diff
@@ -7,6 +7,9 @@ torch==1.4.0
 torchvision==0.5.0
 
 # For Testing
-nose==1.3.7
-coverage==4.4.1
+nose
```

Comment:
Should we consider setting a minimum version for these packages at least? In many ways, I'm in favor of making the version or version ranges explicit. It adds a bit of a maintenance headache because one will need to update it periodically, but on the other hand, it will prevent a hard to find error when these packages change in a way that breaks what we're doing. It's especially problematic, in my experience, with Python packages that seem to have a really short deprecation window before they yank functionality ... looking at you Numpy!

Reply:
I think version ranges are probably the way to go.

Reply:
Agree. Much better than a single version if that's possible Gives a bit of flexibility.